From dc2cd6782f834a75d751d7529eff47c89b3658a9 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:41:34 -0400
Subject: [alsa] hda/hdmi: set depop_delay for haswell plus

Message-id: <140972823573-427-patchset-jkysela@redhat.com>
Patchwork-id: 91123
O-Subject: [RHEL 7.1 PATCH 426/442] ALSA: hda/hdmi - set depop_delay for haswell plus
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Both Haswell and Broadwell need set depop_delay to 0. So apply this
setting to haswell plus.

Signed-off-by: Libin Yang <libin.yang@intel.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: d35f64e748e7752a5a60b1c7798cece51d19a213)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/patch_hdmi.c b/sound/pci/hda/patch_hdmi.c
index 36badba..5e229f7 100644
--- a/sound/pci/hda/patch_hdmi.c
+++ b/sound/pci/hda/patch_hdmi.c
@@ -2330,9 +2330,8 @@ static int patch_generic_hdmi(struct hda_codec *codec)
   intel_haswell_fixup_enable_dp12(codec);
  }
 
- if (is_haswell(codec) || is_valleyview(codec)) {
+ if (is_haswell_plus(codec) || is_valleyview(codec))
   codec->depop_delay = 0;
- }
 
  if (hdmi_parse_codec(codec) < 0) {
   codec->spec = NULL;
-- 
1.7.1